OpenNefia.Content/Resources/Locale/de_DE/Elona/Common.lua

```diff
@@ -0,0 +1,14 @@
+Elona.Common = {
+    Quotes = function(str)
+        return ("\"%s\""):format(str)
+    end,
+    ItIsImpossible = "It's impossible.",
+    NothingHappens = "Nothing happens...",
+    SomethingIsPut = "Something is put on the ground.",
+    PutInBackpack = function(_1)
+        return ("You put %s in your backpack."):format(_1)
+    end,
+    NameWithDirectArticle = function(entity)
```

Comment:
This also needs to be added to `en_EN` and `ja_JP` with a call to `_.name(entity)` instead.

Reply:
added in 0845470